code/game/objects/items/granters.dm

```diff
@@ -523,6 +526,32 @@
 		desc = "It's a damaged upgrade module."
 		name = "damaged board"
 
+/obj/item/book/granter/martial/worldshaker
```

Comment:
something i don't like is having both preternis martial arts at the same time. should probably remove the other one